conda/utils.py

```diff
@@ -17,10 +17,43 @@
 from .auxlib.compat import Utf8NamedTemporaryFile, shlex_split_unicode
 from .common.compat import isiterable, on_win
 from .common.url import path_to_url
+from .deprecations import deprecated
 
 log = logging.getLogger(__name__)
 
 
+@deprecated(
+    "25.3",
+    "26.3",
+    addendum="Use `conda.common.path.unix_path_to_win` instead.",
```

Comment:
Is there a reason we can't replace this with "from conda.common.path import unix_path_to_win"?

Reply:
The implementation is a little bit different, and the output is a bit different. In particular the windows drive letter case is always lower case in this function. But not for `conda.common.path.unix_path_to_win`.

ref: https://github.com/conda/conda/pull/15191